File: KPI_2019-09-11/adminsystem/php/Example/jsp/Sample Source/src/com/lit/litgenerator/LitDecode.java
package com.lit.litgenerator;

import java.util.HashMap;

public class LitDecode {
	private HashMap<String, String> hmVal = null; 
	
	public LitDecode(String pDecodeStr) {
		hmVal = new HashMap<String, String>();
		pDecodeStr = pDecodeStr.substring(pDecodeStr.indexOf("{")+1 , pDecodeStr.indexOf("}")-1);
		pDecodeStr = pDecodeStr.replaceAll("\"", "");
		
		String[] aVal1 = pDecodeStr.split(",");
		String sKey = "", sVal = "";
		for(int i=0; i<aVal1.length; i++) {
			sKey = aVal1[i].substring(0, aVal1[i].indexOf(":"));
			sVal = aVal1[i].substring(aVal1[i].indexOf(":")+1);
			try {
			sVal = sVal.replace("\\", "");
			} catch (Exception e) {
				System.out.println("");
			}
			if(!"".equals(sKey))
				hmVal.put(sKey, sVal); 
		}
	}

	public String getVal(String pIdenVal) {
		return hmVal.get(pIdenVal);
	}
}

File: KPI_2019-09-11/adminsystem/php/Example/asp/Sample Source/App_Code/LitDecode.cs
using System;
using System.Collections.Generic;
using System.Web;
using System.Collections;
/// <summary>
/// LitDecode의 요약 설명입니다.
/// </summary>
public class LitDecode
{
    private Hashtable htVal = null;

    public LitDecode(string pDecodeStr) {
		htVal = new Hashtable();
        pDecodeStr = pDecodeStr.Substring(pDecodeStr.IndexOf("{") + 1, pDecodeStr.LastIndexOf("}") - 1);
		pDecodeStr = pDecodeStr.Replace("\"", "");
		
		string[] aVal1 = pDecodeStr.Split(new Char [] {','});
		string sKey = "";
        string sVal = "";
		for(int i=0; i<aVal1.Length; i++) {
			//aVal1[i] = aVal1[i].replaceAll("\"", "");
			sKey = aVal1[i].Substring(0, aVal1[i].IndexOf(":"));
			sVal = aVal1[i].Substring(aVal1[i].IndexOf(":")+1);
			try {
			sVal = sVal.Replace("\\", "");
			} catch (Exception e) {
			}
			if(!((string)"").Equals(sKey))
                htVal.Add(sKey, sVal);
		}
	}

    public string getVal(string pIdenVal)
    {
        return (string)htVal[pIdenVal];
    }

}